packages/run-protocol/src/vaultFactory/vaultFactory.js

```diff
@@ -146,27 +146,28 @@ export const start = async (zcf, privateArgs) => {
     return vm;
   };
 
+  const makeLoanHook = async seat => {
```

Comment:
for my edification, why was this moved?

requested change: the type annotation should move with it,
```
/** @param {ZCFSeat} seat */
```

Reply:
@dtribble  said a couple of times that he liked to move code to higher levels to reduce indentation. Since this isn't using anything from inside `makeVaultInvitation`, this is a safe transform.

Reply:
Code taking more vertical space means less on the screen at a time, so less context. Our narrow lines means that each level of indentation has a chance of leading to more lines wrapping and thus les vertical content.